distribution/product/modules/distribution/LICENSE.txt

```diff
@@ -16,167 +16,166 @@ Name
 _________________________________________________________________________________________________________________________________________________
 pdepublishing-ant.jar                                                                               jar            epl1
 pdepublishing.jar                                                                                   jar            epl1
-org.wso2.carbon.server.admin.stub_4.7.0.beta10.jar                                                  bundle         apache2
 org.eclipse.equinox.p2.jarprocessor_1.0.300.v20131211-1531.jar                                      bundle         epl1
-org.wso2.carbon.authenticator.stub_4.7.0.beta10.jar                                                 bundle         apache2
 org.eclipse.equinox.preferences_3.6.0.v20160120-1756.jar                                            bundle         epl1
-org.wso2.carbon.securevault_4.7.0.beta10.jar                                                        bundle         apache2
-org.wso2.carbon.crypto.impl_1.1.9.jar                                                               bundle         apache2
 org.wso2.orbit.sun.xml.bind.jaxb_2.3.2.wso2v1.jar                                                   bundle         apache2
-javax.cache.wso2_4.7.0.beta10.jar                                                                   bundle         apache2
-org.wso2.carbon.server.admin.ui_4.7.0.beta10.jar                                                    bundle         apache2
-org.wso2.carbon.registry.core_4.7.0.beta10.jar                                                      bundle         apache2
+org.wso2.carbon.roles.mgt.ui_4.7.0.rc1.jar                                                          bundle         apache2
```

Comment:
these should bot be rc1.jar. All the jars should be `4.7.0.jar`

Reply:
fix in other places too

Reply:
Updated